### ISSUE TITLE
Consider removing the arguments field of BoefjeMeta

### ISSUE THREAD

**Issue by Donnype** [minvws/nl-kat-coordination#763]
Only the hydrated input ooi is added and used in this field, so a generic dict (or at least with the name "arguments"...) to which we only add an "input" key at runtime is not a nice solution for the way we currently run plugins.

**Comment by underdarknl** [comment]
I dont think this holds true, since we also want the env+arugments to be stored, hashed and visible for our paper trail.
